docs/topics/proc_interacting-with-the-circuit-breaker-booster.adoc

```diff
@@ -41,7 +41,8 @@ If the remote endpoint is unavailable, the `name` service responds with an HTTP
 * the `/api/cb-state` endpoint, which returns the state of the Circuit Breaker.
 The state can be:
 ** _open_ : the circuit breaker is preventing requests from reaching the failed service,
-** _closed_: the circuit breaker is allowing requests to reach the service.
+** _closed_: the circuit breaker is allowing requests to reach the service,
+** _half open_: the circuit breaker is allowing a request to reach the service. If the request succeeds, the state of the service is reset to open. If the request fails, the timer is restarted.
```

Comment:
While circuit breakers can be in the half-open state, the `/api/cb-state` endpoint actually only ever returns `open` or `closed`.

Reply:
Good point. Thank you!